### ISSUE TITLE
REPEAT service in a messaging loop with a station sending ack/rej messages

### ISSUE THREAD

**Issue by hessu** [hemna/aprsd-repeat-plugins#1]
The REPEAT service is sending a lot of repeated messages to some stations, for example at as shown here: https://www.facebook.com/photo.php?fbid=10161120164983395&set=p.10161120164983395&type=3 

It seems like REPEAT is not handling ack/rej messages sent by these stations, and treating them as invalid commands.

<img width="853" alt="Screenshot 2023-01-14 at 11 10 18" src="https://user-images.githubusercontent.com/1135075/212464992-638421b2-ca8c-42b2-b1fe-b0f9d1f2b487.png">
<img width="663" alt="Screenshot 2023-01-14 at 11 10 05" src="https://user-images.githubusercontent.com/1135075/212464993-3d3ad590-4178-4f99-bdea-29536abc8079.png">

Please make sure message identifiers of _up to 5 **alphanumeric** characters_ are handled, as per APRS101.PDF section 14. Thanks.

Please also consider having a rate limit of some sort to prevent indefinite loops, if for example another bot accidentally messages your bot.

**Comment by hemna** [comment]
Thanks for the bug.  It looks like the library that aprsd is using is not parsing out reject messages, which is a new message/ack/reject to me.   

```
└─> python decode.py
Raw = 'HB9FDL-1>APK102,HB9FM-4*,WIDE2,qAR,HB9FEF-11::REPEAT   :rej4139'
message
{'raw': 'HB9FDL-1>APK102,HB9FM-4*,WIDE2,qAR,HB9FEF-11::REPEAT   :rej4139', 'path': ['HB9FM-4*', 'WIDE2', 'qAR', 'HB9FEF-11'], 'via': 'HB9FEF-11', 'addresse': 'REPEAT', 'format': 'message', 'response': 'rej', 'msgNo': '4139', 'raw_dict': {'raw': 'HB9FDL-1>APK102,HB9FM-4*,WIDE2,qAR,HB9FEF-11::REPEAT   :rej4139', 'from': 'HB9FDL-1', 'to': 'APK102', 'path': ['HB9FM-4*', 'WIDE2', 'qAR', 'HB9FEF-11'], 'via': 'HB9FEF-11', 'addresse': 'REPEAT', 'format': 'message', 'response': 'rej', 'msgNo': '4139'}, 'from_call': 'HB9FDL-1', 'to_call': 'REPEAT', 'packet_type': 'message'}


MessagePacket(from_call='HB9FDL-1', to_call='REPEAT', addresse='REPEAT', format='message', msgNo='4139', packet_type='message', timestamp=1673704308, raw='HB9FDL-1>APK102,HB9FM-4*,WIDE2,qAR,HB9FEF-11::REPEAT   :rej4139', path=['HB9FM-4*', 'WIDE2', 'qAR', 'HB9FEF-11'], via='HB9FEF-11', message_text=None)
```

**Comment by hemna** [comment]
Have a patch to support reject messages.

https://github.com/craigerl/aprsd/pull/109

**Comment by hemna** [comment]
This has been fixed with the release of aprsd 3.0.2 and upgrade of the repeat service using the latest.

**Comment by hessu** [comment]
Thank you!
